python/test/unit/language/test_core.py

```diff
@@ -2284,7 +2304,15 @@ def test_scan_layouts(M, N, src_layout, axis, device):
 @pytest.mark.parametrize("reduce_op", ["sum", "max"])
 def test_reduce_layouts(M, N, src_layout, axis, reduce2d, dtype_str, reduce_op, device):
     if is_hip():
-        pytest.skip("TODO test_reduce_layouts is not supported in HIP")
+        if  isinstance(src_layout, MfmaLayout) and ((N == 16 and src_layout.instr_shape[1] > 16) or (M == 16 and src_layout.instr_shape[0] > 16)):
+            pytest.skip("Tensor dimensions are incompatible with MFMA layout")
+        if isinstance(src_layout, MfmaLayout) and src_layout.is_transposed and axis == 0:
+            pytest.skip("Reduce along axis 0 is not supported in transposed mfma layout")
+        if reduce_op == "max" and (dtype_str == "float16" or dtype_str == "float32") :
```

Comment:
Can you double check if max is still broken? Last time I checked it was working.

Reply:
You are right, it seems that LLVM version was updated in the meantime. I updated the code accordingly.